src/test/java/com/purbon/kafka/topology/integration/AccessControlManagerIT.java

```diff
@@ -38,7 +38,7 @@
 
   private static AdminClient kafkaAdminClient;
   private AccessControlManager accessControlManager;
-  private ClusterState cs;
+  private ClusterStateManager cs;
```

Comment:
suggestion, whey not using instead of _cs_ as variable name, something like _clusterStateManager_ ? what do you think? might be _csm_ ?